zoo-hope/src/data/nav.ts

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+export const NavBarData = () => {
+    
+const elements = [
+    { name: "Про нас", url: "/" },
+    { name: "Контакти", url: "/" },
+    {
+      name: "Звітність",
+      Ielements: [
+        { name: "DropDownLink1", url: "/" },
+      ],
+    },
+    {
+      name: "Шукають дім",
+      Ielements: [
+        { name: "DropDownLink1", url: "/" },
+      ],
+    },
+    {
+      name: "Лікування",
+      Ielements: [
+        { name: "DropDownLink1", url: "/" },
+      ],
+    },
+    {
+      name: "Стерилізація",
+      Ielements: [
+        { name: "DropDownLink1", url: "/" },
+      ],
+    }]
+
+    const [clickedButtonUA, setClickedButtonUA] = useState(true);
```

Comment:
This sections are not necessary to more out
return please useState back
Threre should be only that:
const elements = [...];
export elements;

Reply:
done